contrib/validation/validation_cases_lucille.txt

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 The Lethe Authors
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+# This file contains the list of validation test cases used in the large-scale validation cases for lethe
+# The cases assume that the validation is ran from the root folder of the lethe source code
+# For every cases, the path of the vase must be specified followed by the number of processors with which
+# the case must be ran. The file allows for comments, which are specified by a "#"
+
+examples/incompressible-flow/2d-lid-driven-cavity 1
+examples/incompressible-flow/2d-taylor-couette 1
```

Comment:
Should we run with more than one process to be sure the parallel functionality is valid?

Reply:
I don't think it's necessary for these two cases :)